sdk/communication/communication-job-router/src/jobRouterClient.ts

```diff
@@ -203,11 +285,18 @@ export class JobRouterClient {
    * Gets the list of jobs.
    * @param options - List jobs options.
    */
-  public listJobs(options: ListJobsOptions = {}): PagedAsyncIterableIterator<RouterJobItem> {
+  public listJobs(options: ListJobsOptions = {}): TransformingPagedAsyncIterableIterator<RouterJobItemGenerated, RouterJobItem> {
```

Comment:
@williamzhao87 This works, and is an equivalent interface to PagedAsyncIterableIterator, it just has to be a different name like this.

Reply:
Great work!